pkg/types/ovirt/config.go

```diff
@@ -0,0 +1,70 @@
+package ovirt
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+)
+
+var defaultOvirtConfigEnvVar = "OVIRT_CONFIG"
+var defaultOvirtConfigPath = filepath.Join(os.Getenv("HOME"), ".ovirt", "ovirt-config.yaml")
+
+// OvirtConfig holds oVirt api access details
+type OvirtConfig struct {
```

Comment:
This type is duplicated elsewhere: https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/ovirt/config.go#L19

This (pkg types) is a better place for it. Would be nice to centralize this as part of this pr.

Reply:
good catch, done.